docs/getting_started.md

```diff
@@ -44,156 +45,167 @@ More resources for learning about `SingleCellExperiment` objects:
+All processed objects include [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html) results found in the {ref}`colData() of the SingleCellExperiment object<sce_file_contents:Singlecellexperiment cell metrics>` or the {ref}`.obs slot of the AnnData object<sce_file_contents:Anndata cell metrics`.
 `miQC` jointly models the proportion of mitochondrial reads and the number of unique genes detected in each cell to calculate the probability of a cell being compromised (i.e., dead or damaged).
 High-quality cells are those with a low probability of being being compromised (< 0.75) or sufficiently low mitochondrial content.
-All cells that are identified as low-quality cells will have `FALSE` in the `miQC_pass` column of the `colData()` and can be removed prior to downstream analyses.
 
-The following command can be used to remove the low-quality cells:
+The probability compromised for each cell as calculated by `miQC` can be found in the `prob_compromised` column of the `colData` of `SingleCellExperiment` objects.
 
 ```r
-# read in the filtered object
-sce <- readRDS("SCPCS000000/SCPCL000000_filtered.rds")
-
-# filter the `SingleCellExperiment`
-# the `$` notation denotes access to the colData slot of the `SingleCellExperiment` object
-filtered_sce <- sce[, which(sce$miQC_pass)]
+# pull out probability compromised for each cell
+processed_sce$prob_compromised
 ```
 
-For more information on using `miQC` for filtering cells, see the following resources:
-- [`miQC` vignette](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html)
-- [Hippen _et al._ 2021](https://doi.org/10.1371/journal.pcbi.1009290)
+The probability compromised for each cell as calculated by `miQC` can be found in the `prob_compromised` column of the `.obs` slot of `AnnData` objects.
 
-You can also directly filter cells based on the number of unique genes, total reads, and fraction of mitochondrial reads.
-We used the function [`scuttle::addPerCellQCMetrics()`](https://rdrr.io/github/LTLA/scuttle/man/addPerCellQCMetrics.html) to calculate these metrics and have included them in the `colData` of the filtered `SingleCellExperiment` objects.
+```python
+# pull out probability compromised for each cell
+processed_adata.obs["prob_compromised"]
+```
 
-The following columns are added by `scuttle::addPerCellQCMetrics()` and can be found in the `colData`:
+Additionally we include a column, `scpca_filter`, that labels cells as either `Keep` or `Remove` based on having both a `prob_compromised` < 0.75 and number of unique genes identified > 200.
+All cells included in the processed object will have `Keep` in the `scpca_filter` column.
+If you prefer to work with the object prior to removal of any low-quality cells, please use the filtered object.
```

Comment:
I don't know if we need a bit of extra info here, just as a reminder of what that object is.
```suggestion
If you prefer to work with the object prior to removal of any low-quality cells, please use the filtered object, which contains all cells that were not discarded as empty droplets.
```